packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -462,26 +461,26 @@ class ServiceConnectionManager {
       registrations.displayRefreshRate,
       args: {'viewId': viewId},
     );
-    final double fps = displayRefreshRateResponse.json['fps'];
+    final double? fps = displayRefreshRateResponse.json!['fps'];
```

Comment:
make this non-nullable, add ! to the end, and remove the ! below

Reply:
done